purview/models/distgit.py

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from neomodel import (
+    StructuredNode, UniqueIdProperty, RelationshipTo, StringProperty,
+    DateTimeProperty)
+
+
+class DistgitCommit(StructuredNode):
```

Comment:
Can you construct a URL back to the commit with the data you have here?  I think you're missing repo name, namespace, etc.

Reply:
The repo name is called `module` in the database. Maybe it's worth renaming it here for clarity. I'll double check on the namespace.

Reply:
@ralphbean so looking at the gitbz code, the namespace is not stored... let's talk tomorrow about workarounds.

Reply:
When namespaces were added, we did not update gitbz (an oversight really, but it was not clear if it's actually used anywhere). It would be really to modify it to store the module as `namespace/name`. Adding another column for the namespace would also be an option.

Reply:
I prefer the idea of a separate namespace column. If we're touching the data model, it would be a good opportunity to add `author_email` and `committer_email` as well.

Reply:
I opened COMPOSE-2409 to track adding that.